WordPress/Classes/ViewRelated/NUX/Signup.storyboard

```diff
@@ -1,7 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> This is currently using the site creatino blogs illustration because I couldn't find the one from the mockups. I can replace it next week.

Would you mind adding a task to the master issue?

Reply:
👍 DONE